solr/core/src/test/org/apache/solr/cloud/SolrCLIZkUtilsTest.java

```diff
@@ -69,6 +70,7 @@ public static void closeConn() {
       zkClient = null;
     }
     zkAddr = null;
+    System.clearProperty("solr.solr.home");
```

Comment:
isn't there a new / better way of handling system properties in tests?  @dsmiley ?   If we are fixing this, maybe use the proper technique?

Reply:
Yeah I was hesitating on doing it this way but ended up following the existing "style".
But I think we don't need this as we can rely on _SystemPropertiesRestoreRule_. 
Will test and let you know.

Reply:
We probably should have a follow up ticket to just sweep through the code and get rid of all the `System.clearProperty` (and maybe add it to policeman) so the next person knows to use _SystemPropertiesRestoreRule_...